examples/menu/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import { Typography } from "@material-ui/core";
+import { CalendarToday, Home, School, Settings } from "@material-ui/icons";
+import { Master, Menu, MenuCollapsibleItem, MenuItemLink } from "@vivid-planet/react-admin-mui";
```

Comment:
Good naming?? Master and Menu are very generic

Reply:
I think we should name it more closely like MUI, because a Menu in MUI is something completely different. Perhaps `MenuDrawer`? 

Same for Master: Maybe `HeaderAppBar`?

Reply:
MenuContainer instead of Menu? Master is to generic, I like daniels example MenuDrawer?

Reply:
Menu -> MenuDrawer sounds great

Master is difficult. It does more than a `HeaderAppBar` suggests. And `MenuDrawer` is already taken :) My suggestion: `DefaultMaster`, or even stick with `Master`

Reply:
What about MasterLayout?

Reply:
MasterLayout :+1: